cnc_ctrl_v1/cnc_ctrl_v1.ino

```diff
@@ -30,21 +30,16 @@ void setup(){
     sys.inchesToMMConversion = 1;
     setupAxes();
     Serial.begin(57600);
-    readyCommandString.reserve(128);           //Allocate memory so that this string doesn't fragment the heap as it grows and shrinks
-    gcodeLine.reserve(128);
+    readyCommandString.reserve(INCBUFFERLENGTH);           //Allocate memory so that this string doesn't fragment the heap as it grows and shrinks
+    gcodeLine.reserve(INCBUFFERLENGTH);
+    Timer1.initialize(LOOPINTERVAL);
+    Timer1.attachInterrupt(runsOnATimer);
     
     Serial.print(F("PCB v1."));
     Serial.print(getPCBVersion());
     Serial.println(F(" Detected"));
-    
+    Serial.println(F("Grbl v1.00"));  // Why GRBL?  Why not Maslow?
```

Comment:
There was a desire for grbl compatibility, so this hardware might run on front ends written for grbl.

Reply:
Universal gcode sender in particular won't start until it has received the command "Grbl v1.00". It's a hack 😒

Reply:
Well that is just silly.  Grbl provides a nice "ready" and "ok" commands for this purpose.  Oh well.